### DISCUSSION TITLE
Actions Caching help doc shows outdated 400MB file size limit

### DISCUSSION THREAD

**Discussion by thisismydesign** [community/community#25052] [Actions]
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows</a></p>
<p>&gt; Individual files in a cache don’t exceed 400 MB. You will receive a 400 - Bad Request if you exceed this limit.</p>
<p>Meanwhile</p>
<p><a href="https://github.com/actions/cache/releases/tag/v1.1.0" rel="nofollow noopener">https://github.com/actions/cache/releases/tag/v1.1.0</a></p>

**Comment by BrightRan** [comment, marked as answer]
<p>The doc about <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy" rel="nofollow noopener">Usage limits of Caching</a> has been updated to the following now:</p>
<p><strong>GitHub will remove any cache entries that have not been accessed in over 7 days. There is no limit on the number of caches you can store, but the total size of all caches in a repository is limited to 2 GB. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 2 GB.</strong></p>
